src/plays/devblog/components/Header.js

```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom';
+
+const Header = () => {
+  return (
+    <header className="flex bg-gray-800 text-white justify-between px-10 py-4 w-full">
+      <Link className="logo text-3xl font-bold" to="/plays/coderamrin/devblog">
+        DevBlog
+      </Link>
+
+      <nav>
+        <a href="https://amrin.onrender.com/">About me</a>
```

Comment:
@Coderamrin Open this link in. anew tab.

Reply:
> @Coderamrin Open this link in a new tab.

@atapas  added the play video